bitbucket/exceptions.py

```diff
@@ -51,4 +55,7 @@ def bitbucket_exception_to_api():
         else:
             logging.exception(f'Unexpected Bitbucket exception: {str(ex)}')
 
-        raise BitbucketBaseAPIException(status_code=ex.response.status_code)
+        if ex.response.status_code == 400:
```

Comment:
minor: as you are already importing it, maybe using the `status.HTTP_400_BAD_REQUEST` here?

Reply:
Much much better. I was too distracted!